masterlist.yaml

```
@@ -831,6 +831,45 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+   ###### Gameplay - Quests ######
+     ### Rise of House Telvanni ###
+  - name: 'Rise of House Telvanni.esm'
```

No issue with added sections, but try to keep the formatting consistent.

```suggestion
###### Gameplay - Quests ######
  ### Rise of House Telvanni ###
  - name: 'Rise of House Telvanni.esm'
```